bin/genesis

```diff
@@ -758,7 +758,23 @@ sub __prompt_for_line {
 	# being an optional error message
 	my $validate;
 	if (defined($validation)) {
-		if ($validation =~ m/^\/(.*)\/(i?m?s?)$/) {
+		if ($validation eq "url") {
+			$validate = sub() {
+				if (is_valid_uri($_[0])) {
+					return "";
```

Comment:
It's more perl-y (and I believe @dennisjbell  will agree, to phrase this as:

```
return "" if is_valid_uri($_[0])
```

Reply:
I think you are right. This is first time I code in perl, so it is easy to be perl-y, or perl-c :)
I will fix this